src/lib/character_vocabulary.cc

```diff
@@ -50,8 +50,10 @@ namespace nmtkit {
 
 CharacterVocabulary::CharacterVocabulary(
     const string & corpus_filename,
+    unsigned unk_frequency,
```

Comment:
Could you add test code in src/test/character_vocabulary_test.cc for this parameter?